README.md

```diff
@@ -3,17 +3,17 @@
 
 ## Table of Contents
 - [Introduction](#introduction)
-- [Getting Started](#getting-started)
-- [Contributing](#contributing)
+- [End Users](#end-users)
```

Comment:
Adding summation of chat @eggshell and I just had:
I believe readability of documentation is paramount. @eggshell agreed.